posts/buffer-analysis-update.md

```diff
@@ -0,0 +1,55 @@
+---
+version: 1.0.0
+title: California forest carbon buffer pool update
+authors:
+  - Grayson Badgley
+  - Danny Cullenward
+date: 11-17-2022
+summary: Offset projects hit by recent wildfires report larger carbon losses than we had projected.
+card: buffer-analysis-update
+data:
+  ACR255:
+    slag:
+      max: 82.69
+      current: 73.24
+      termination: 62.84
+    carbonplan:
```

Comment:
Should we remove these if we aren't going to use in the table? Alternatively, we could keep the definitions here and thread through to the `ReversalsTable` via props.